content/docs/guides/metrics-api.md

```diff
@@ -0,0 +1,408 @@
+| -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ----------------- |
+| [Account-level cumulative metrics](https://api-docs.neon.tech/reference/getconsumptionhistoryperaccount) | Aggregates all metrics from all projects in an account into a single cumulative number for each metric                   | Scale plan only   |
+| [Granular project-level metrics](https://api-docs.neon.tech/reference/getconsumptionhistoryperproject)   | Provides detailed metrics for each project in an account at a specified granularity level (e.g., hourly, daily, monthly) | Scale plan only   |
+| [Billing period project-level metrics](https://api-docs.neon.tech/reference/listprojectsconsumption)     | Offers consumption metrics for each project in an account for the current billing period                                 | All plans         |
+| [Single project metrics](https://api-docs.neon.tech/reference/getproject)                                | Retrieves detailed metrics and quota information for a specific project                                                  | All plans         |
+
+## Get account-level aggregated metrics
+
+Using the [Get account consumption metrics API](https://api-docs.neon.tech/reference/getconsumptionhistoryperaccount)), you can find total usage across all projects in your organization. This provides a comprehensive view of consumption metrics accumulated for the billing period.
+
+Here is the URL in the Neon API where you can get account-level metrics:
+
+```bash
+https://console.neon.tech/api/v2/consumption_history/account
+```
+
+This API endpoint accepts the following query parameters: `from`, `to`, `granularity`, `org_id`, and `include_v1_metrics`.
+
+### Choosing your account
+
+Include the unique `org_id` for your organization to retrieve account metrics for that specific organization. If not specified, metrics for your personal account will be returned.
+
+<Admonition type="note">
+Organizations are currently in private preview. For more information about this upcoming feature, see [Organizations](/docs/manage/organizations).
+</Admonition>
+
+### Set a date range for granular results
+
+You can set `from` and `to` query parameters, plus a level of granularity to define a time range that can span across multiple billing periods.
+
+- `from` — Sets the start date and time of the time period for which you are seeking metrics.
+- `to` — Sets the end date and time for the interval for which you desire metrics.
+- `granularity` — Sets the level of granularity for the metrics, such as `hourly`, `daily`, or `monthly`.
+
+The response is organized by periods and consumption data within the specified time range.
+
+[Details on setting a date range](#details-on-setting-a-date-range)
```

Comment:
Does this link need some context, like "For information about ..., see x."

Reply:
Yes, changed.